Gemfile

```diff
@@ -50,7 +50,7 @@ gem 'decision_trees', '~> 2.0.0'
 gem 'feedback', '~> 0.3.0'
 gem 'mortgage_calculator', '~> 1.3.5'
 gem 'pensions_calculator', '~> 0.2'
-gem 'rio', git: 'git@github.com:moneyadviceservice/rio.git', ref: 'bb267ca11932fd7958077d89f9802c9ae2d60d5b'
+gem 'rio', git: 'git@github.com:moneyadviceservice/rio.git', ref: '35fc40dd3adbd31124661ea66278fdad3bd6c23b'
```

Comment:
Thanks.

The purpose of Gemfile and Gemfile.lock is a bit different. Generally speaking, in the Gemfile you should state a minimal set of requirements that should be met by your libraries. On the other hand, Gemfile.lock is meant to specify working configuration with all dependencies resolved (and usually is autogenerated).

I think that in this case, it might be a good idea to remove `ref` option from `Gemfile` and use `bundle update` when we want to update rio gem version.